xbmc/settings/dialogs/GUIDialogContentSettings.h

```diff
@@ -76,6 +74,10 @@ class CGUIDialogContentSettings : public CGUIDialogSettingsManualBase
   void InitializeSettings() override;
 
 private:
+  void setLabel2(const std::string &settingid, const std::string &label);
+  void toggleState(const std::string &settingid, bool enabled);
+  void setFocus(const std::string &settingid);
```

Comment:
Because they are private? 
I think I would have done something similiar, due to my use of other languages.
If it's not wanted, it might be worth a note here http://kodi.wiki/view/Official:Code_guidelines_and_formatting_conventions#Methods